
running 3 tests
test test1 ... ok
test test2 ... ok
test test3 ... FAILED

failures:

---- test3 stdout ----
thread 'test3' panicked at 'this should not run', ui/test-attrs/test-filter-multiple.rs:16:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    test3

test result: FAILED. 2 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

